test/parallel/test-internal-util-decorate-error-stack.js

```diff
@@ -32,7 +32,7 @@ function checkStack(stack) {
 }
 let err;
 const badSyntaxPath =
-    path.join(common.fixturesDir, 'syntax', 'bad_syntax')
+    fixtures.path('syntax', 'bad_syntax')
```

Comment:
nit: this line can be merged with the line above now. This can be handled during landing